tests/components/test_component_extract_subset.cpp

```diff
@@ -0,0 +1,8 @@
+#include <wmtk_components/extract_subset/internal/extract_subset_2d.hpp>
+#include <catch2/catch_test_macros.hpp>
+
+
+TEST_CASE("manual test case", "[components][extract_subset][2D]")
```

Comment:
Test cases should not have spaces in their name, use `_` instead: `manual_test_case`.

Furthermore, test case names should be very descriptive because all test cases share the same scope.